### PR TITLE
[BugFix] Fix inconsistent http method since BE had used POST for transaction action

### LINKED ISSUE #4757: Transaction interface for stream load
## Feature request

**Is your feature request related to a problem? Please describe.**

When the current flink-starrocks-connector implements `exactly once` semantics, it needs to save the data generated in a single checkpoint cycle, write it to Flink storage during checkpoint, and then write it to StarRocks, which leads to extra memory and IO read and write overhead while very easy to OOM

**Describe the solution you'd like**

We plan to provide a transaction interface for stream load to facilitate two-phase commit across systems with the client.

**Describe alternatives you've considered**

### start transaction
``` bash
curl -H "label:${label}"
    -X PUT http://fe_host:http_port/api/{db}/transaction/begin
```

### commit transaction
``` bash
curl -H "label:${label}"
    -X PUT http://fe_host:http_port/api/{db}/transaction/commit
```
### rollback transaction
```bash
curl -H "label:${label}"
    -X PUT http://fe_host:http_port/api/{db}/transaction/rollback
```
### send data
```bash
curl -H "label:${label}" 
    -T /path/to/data.csv
    -X PUT http://fe_host:http_port/api/{db}/transaction/load/{table}
```
> user can send data `multi times` in one transaction using both `mini batch` and `stream` mode

### list transaction
```bash
curl -H "label:${label}"
    -X PUT http://fe_host:http_port/api/{db}/transaction/list

### PR DESCRIPTION
**Pull request by meegoo**
…saction action

## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #4757

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ public TransactionLoadAction(ActionController controller) {
 
     public static void registerAction(ActionController controller) throws IllegalArgException {
         TransactionLoadAction ac = new TransactionLoadAction(controller);
-        controller.registerHandler(HttpMethod.PUT, "/api/{" + DB_KEY + "}/transaction/{" + TXN_OP_KEY + "}", ac);
+        controller.registerHandler(HttpMethod.POST, "/api/{" + DB_KEY + "}/transaction/{" + TXN_OP_KEY + "}", ac);
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java:60]: why we put transaction under a database?
why not /api/transaction/begin?

**Comment by chaoyli** [comment]
run starrocks_fe_unittest

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/rest/TransactionLoadAction.java|1|1|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java (hunk 1 above)
Comment (meegoo) [reply fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java:60]: Implement in other PR

**Comment by meegoo** [comment]
run starrocks_be_unittest

**Review by Astralidea — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Comment by meegoo** [comment]
@mergify backport branch-2.3

**Pull request merged**